source/_components/switch.snmp.markdown

```diff
@@ -10,12 +10,12 @@ footer: true
 logo: network-snmp.png
 ha_category: Switch
 ha_iot_class: "Local Polling"
-ha_release: 0.57
+ha_release: 0.70.1
```

Comment:
Please don't change it. With `ha_release` we track when the platforms/components were introduced.